source/_components/sensor.filesize.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: "file size sensor"
+description: "Component for monitoring the size of a file."
+date: 2018-02-06 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: file.png
+ha_category: Sensor
+ha_iot_class: "Local Polling"
+ha_release: 0.63
+---
+
+Component for displaying the size of a file.
+Add to your config:
+
+```yaml
+sensor:
+  - platform: filesize
+    file_paths:
+      - /config/home-assistant_v2.db
+  ```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
Sure! Btw I don't think this rule is being applied consistently by reviewers? See [example](https://github.com/dgomes/home-assistant.github.io/blob/d9186d325e9c54c322a8bec9e29fea27220e3bbb/source/_components/sensor.sql.markdown)

Reply:
It is consistent. If you take a look at that PR, you would have seen my RFC for the same thing as well (which I did before you made the above comment).

Secondly, you need to remove the "Configuration variables:" header as well.